solutions/secure/variables.tf

```diff
@@ -58,8 +58,8 @@ variable "members" {
 
 variable "member_memory_mb" {
   type        = number
-  description = "The memory per member that is allocated. For more information, see https://cloud.ibm.com/docs/databases-for-elasticsearch?topic=databases-for-elasticsearch-resources-scaling"
-  default     = 1024
+  description = "Allocated memory per member. [Learn more](https://cloud.ibm.com/docs/databases-for-elasticsearch?topic=databases-for-elasticsearch-resources-scaling)"
+  default     = 4096
```

Comment:
I guess `member_host_flavor` should also be exposed in the DA?

Reply:
Added in latest 3414351